util/info.c

```diff
@@ -114,11 +114,14 @@ static int str2cap(char *inputstr, uint64_t *value)
 	ORCASE(FI_RMA);
 	ORCASE(FI_TAGGED);
 	ORCASE(FI_ATOMIC);
+	ORCASE(FI_ATOMICS);
+	ORCASE(FI_MULTICAST);
 
 	ORCASE(FI_READ);
 	ORCASE(FI_WRITE);
 	ORCASE(FI_RECV);
 	ORCASE(FI_SEND);
+	ORCASE(FI_TRANSMIT);
```

Comment:
Yes I noticed this. Since we accept both (in source), shouldn't we match that behavior on the command line? Or is it too late to remove the aliases? I don't understand why they are needed or what value they provide..